requirements.txt

```diff
@@ -1,12 +1,12 @@
-Django==2.0.8
+Django==2.1.7
 celery==4.2.1
 django-authtools==1.6.0
 django-model-utils==3.1.2
 django-solo==1.1.3
-eth-abi==1.2.0
+eth-abi==1.3.0
 eth-utils==1.2.2
 ethereum==2.3.2
 factory-boy==2.11.1
 requests==2.18.4
 urllib3==1.22
-web3[tester]==4.7.2
+web3[tester]==4.8.2
```

Comment:
Maybe we can use web3 4.9.2

Reply:
Good catch .), I will test it and commit an update